eclipse-collections-code-generator/src/main/resources/impl/list/immutable/immutablePrimitiveArrayList.stg

```diff
@@ -53,6 +53,10 @@ import org.eclipse.collections.impl.list.mutable.primitive.<name>ArrayList;
 import org.eclipse.collections.impl.set.mutable.primitive.<name>HashSet;
 import org.eclipse.collections.impl.tuple.primitive.PrimitiveTuples;
 import org.eclipse.collections.impl.utility.Iterate;
+<if(primitive.specializedStream)>
+import java.util.Spliterator;
+import java.util.Spliterators;<endif>
+
 
```

Comment:
Two newlines in a row. This may be a CheckStyle violation, not sure.

Reply:
it doesn't. when i just left one line it gave me a checkstyle warning. please note that in the compiled file just one line remains

Reply:
Even in lists without a specialized stream? I would have thought you'd get the proper formatting by moving the newline from just after to just before the endif.